lib/index.js

```diff
@@ -67,15 +68,18 @@ Parsely.prototype.page = function(page) {
   if (this.options.inPixelMetadata) {
     var aliasedProps = page.properties(this.options.customMapping);
     var metadata = {
-      articleSection: aliasedProps.articleSection || page.category() || aliasedProps.category,
-      thumbnailUrl: aliasedProps.thumbnailUrl || aliasedProps.imageUrl,
-      dateCreated: aliasedProps.dateCreated || aliasedProps.created,
-      headline: aliasedProps.headline,
-      keywords: aliasedProps.keywords,
-      creator: aliasedProps.creator || aliasedProps.author,
-      url: aliasedProps.url
+      section: aliasedProps.section || page.category(),
+      image_url: aliasedProps.image_url || aliasedProps.imageUrl,
+      pub_date: aliasedProps.pub_date || aliasedProps.created,
+      title: aliasedProps.title || page.title(),
+      tags: aliasedProps.tags,
+      authors:  aliasedProps.authors,
+      link: aliasedProps.link || page.url(),
+      page_type: aliasedProps.page_type || 'post'
     };
 
+    if (!is.array(metadata.authors)) metadata.authors = [metadata.authors];
+
```

Comment:
the `tags` field takes an array in the same manner as `authors`, e.g. `"tags": ["yeezus", "college dropout", "808s and heartbreak"]`